turbopack/crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -193,7 +194,20 @@ where
 }
 
 fn create_semaphore() -> tokio::sync::Semaphore {
-    tokio::sync::Semaphore::new(256)
+    // the semaphore isn't serialized, and we assume the environment variable doesn't change during
+    // runtime, so it's okay to access it in this untracked way.
+    static NEXT_TURBOPACK_IO_CONCURRENCY: LazyLock<usize> = LazyLock::new(|| {
```

Comment:
looks like create_semaphore is called just once when creating a DiskFilesystem... maybe don't bother with the lazy lock?